README.md

```diff
@@ -0,0 +1,48 @@
+# api-sdk-manager-go-library
+
+Go implementation of the api-sdk-manager-java-library. Users of the SDK will interact with the SDK manager, 
+which wraps up some useful functionality that determines whether to initialise an OAuth2 or API Key-authenticated
+http client.
```

Comment:
Can you add a brief description of what this service is (rather than only state that it is an implementation of a java service)?